src/components/inputs/PaperInputs.tsx

```
@@ -0,0 +1,167 @@
+			hideSub.remove();
+		};
+	}, [onBlur]);
+
+	return (
+		<Controller
+			control={control}
+			name={name}
+			render={({
+				field: { onChange: controllerOnChange, onBlur: controllerOnBlur, value: controllerValue },
+			}) => {
+				const labelTranslateY = useMemo(() => new Animated.Value(controllerValue ? 0 : 20), []);
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)